community/knative/charts/serving/templates/ingressgateway-deployment.yaml

```diff
@@ -100,7 +100,7 @@ spec:
           valueFrom:
```

Comment:
Looks like we missed the apiVersion on this one from the last PR.  Should be `apps/v1`

Reply:
@bgower-ibm Should the other appVersions here also become `apps/v1` or are they okay to remain `v1`?

Reply:
I tried an install with the apiVersion `apps/v1` got the following error:
```
Error: release my-release failed: Deployment.apps "knative-ingressgateway" is invalid: spec.template.spec.containers[0].env[3].valueFrom.fieldRef.fieldPath: Invalid value: "metadata.name": error converting fieldPath: unsupported pod version: apps/v1
```
Adding `apiVersion: v1` has successfully installed

Reply:
I'm confused, let's make sure we are talking about the same apiVersion.  I'm talking about the apiVersion for `kind: Deployment` like this one:
https://github.com/IBM/charts/blob/master/community/knative/charts/build/templates/controller-pod.yaml#L1

So are you saying you updated this line to `apps/v1` and it didn't work?
https://github.com/IBM/charts/blob/master/community/knative/charts/serving/templates/ingressgateway-deployment.yaml#L1

The only thing I can think of is that the current apiVersion is `extensions/v1beta1` which is listed as an `Old API` in the kube docs.  I wouldn't be surprised if its an early version of the Deployment spec that is structurally different than the GA version (apps/v1).

Reply:
@tomleah :  Sorry forgot to tag you; see above comment.

Reply:
My mistake @bgower-ibm, I was looking in the wrong place.. have committed the change now